### ISSUE TITLE
Pose estimation stuck at nearly identical predictions

### ISSUE THREAD

**Issue by NabilSNaS** [DeepLabCut/DeepLabCut#3074]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Operating System

Linux (Google Colab)

### DeepLabCut version

DLC 3.0.0rc11

### What engine are you using?

pytorch

### DeepLabCut mode

single animal

### Device type

Colab T4- High RAM

## Issue: Coordinates Not Changing in Labeled Videos

I am trying to get the coordinates for the videos I have, but I am noticing that the detected coordinates **do not change much** between frames.

### Steps I Took

1. I have **6 videos for training** and **6 videos to analyze**.
2. Extracted **20 frames from each video**, resulting in **120 frames total** in the training dataset.
3. After labeling the frames locally and training on Google Colab, the output coordinates for the videos **remain almost identical** throughout the entire video.
4. Even when using the **`create_labeled_video`** feature, the labels **do not move** and instead **stay fixed in a corner**.

### Observations

* The same coordinates are being repeated for the whole video.
* This happens regardless of whether the labeling was done locally or the training was done in Colab.
* The problem persists even after generating labeled videos.

### Expected Behavior

The coordinates should **change frame-by-frame** according to object movement in the video.

### Actual Behavior

* Coordinates remain almost constant.
* Labels in the `create_labeled_video` output remain fixed in a corner without tracking movement.

Can you please suggest me how to solve this problem, what is the possible cause here can you help me to figure out?

### YAML Configuration

```yaml
SuperAnimalConversionTables: null
Task: DLC_NewProject
TrainingFraction:
- 0.95
alphavalue: 0.7
batch_size: 8
bodyparts:
- Nose
- Neck
- Front_Right
- Front_Left
- Rear_Right
- Rear_Left
- Back
- Center
- Tail_Base
- Tail_Mid
- Tail_Tip
colormap: rainbow
corner2move2:
- 50
- 50
cropping: false
date: May18
default_augmenter: default
default_net_type: resnet_50
detector_batch_size: 1
detector_snapshotindex: -1
dotsize: 12
engine: pytorch
identity: null
iteration: 0
move2corner: true
multianimalproject: false
numframes2pick: 20
pcutoff: 0.6
project_path: /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18
scorer: Hafija
skeleton:
- - Nose
  - Neck
- - Neck
  - Front_Right
- - Neck
  - Front_Left
- - Front_Right
  - Rear_Right
- - Front_Left
  - Rear_Left
- - Neck
  - Back
- - Back
  - Center
- - Center
  - Tail_Base
- - Tail_Base
  - Tail_Mid
- - Tail_Mid
  - Tail_Tip
- - Rear_Right
  - Rear_Left
skeleton_color: black
snapshotindex: -1
start: 0
stop: 1
video_sets:
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/3_21.mp4:
    crop: 0, 1920, 0, 1080
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/3_22.mp4:
    crop: 0, 1920, 0, 1080
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/3_23.mp4:
    crop: 0, 1920, 0, 1080
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/3_24.mp4:
    crop: 0, 1920, 0, 1080
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/3_25.mp4:
    crop: 0, 1920, 0, 1080
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/3_26.mp4:
    crop: 0, 1920, 0, 1080
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/cage 7-1.mp4:
    crop: 0, 3200, 0, 2400
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/cage 7-2.mp4:
    crop: 0, 3200, 0, 2400
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/cage 7-3.mp4:
    crop: 0, 3600, 0, 2400
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/cage 8-1.mp4:
    crop: 0, 3200, 0, 2400
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/cage 8-2.mp4:
    crop: 0, 3200, 0, 2400
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/cage 8-3.mp4:
    crop: 0, 3600, 0, 2400
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/cage 9-1.mp4:
    crop: 0, 3600, 0, 2400
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/cage 9-2.mp4:
    crop: 0, 3200, 0, 2400
  /content/drive/MyDrive/DLC_VIDEOS/DLC_NewProject-Hafija-2025-05-18/videos/cage 9-3.mp4:
    crop: 0, 3200, 0, 2400
x1: 0
x2: 640
y1: 277
y2: 624
```

---

I have extracted the frames in this way:
```python
deeplabcut.extract_frames(config_path, mode='automatic', algo='kmeans', crop=False)
deeplabcut.extract_frames(
    config_path,
    mode='automatic',
    algo='uniform',
    userfeedback=False,
    crop=False
)
```

create train dataset and train with this code:
```python
deeplabcut.create_training_dataset(config_path)

deeplabcut.train_network(
    config_path,
    displayiters=200,
    saveiters=1000,
    maxiters=20000
)
```

Here is an output frame where you can see the mouse is not labeled correctly and the coordinates are almost constant on that region. 
<img width="2048" height="1535" alt="Image" src="https://github.com/user-attachments/assets/0b400723-b50f-4cf7-9675-c1eff88563fe" />

### Relevant log output

```shell

```

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by maximpavliv** [comment]
Hi @NabilSNaS ,

Thanks for opening the issue!

- What is the model architecture you choose? Can you share your _pytorch_config.yaml_ content please? 
- How did the training go? Can you share your _train.txt_ file so I can have a look as well?
- What are your model's [evaluation](https://deeplabcut.github.io/DeepLabCut/docs/standardDeepLabCut_UserGuide.html#h-evaluate-the-trained-network) metrics? Can you please share them too?
- In the predicted poses CSV file (for the video you analyzed), what are the values of the likelihoods? What `pcutoff` did you select when analyzing the video?

**Comment by NabilSNaS** [comment]
Thanks for your response. 
I chose ResNet-50 as the model. From the logs of training and evaluation, it looks like the model did not learn anything. I wonder if this label alignment issue or training issue. 


Pytorch_config.yaml:
[pytorch_config.yaml](https://github.com/user-attachments/files/21842775/pytorch_config.yaml)

Here is train.txt:

[train.txt](https://github.com/user-attachments/files/21842783/train.txt)


The pcutoff value is 0.6. 

Here is evaluation logs:
```
Evaluation scorer: DLC_Resnet50_DLC_NewProject_GRP3_HealthyAug15shuffle1_snapshot_best-10
DEBUG: No prefix mismatch, loading original weights

train rmse            1301.84
train rmse_pcutoff        NaN
train mAP                0.00
train mAR                0.00
test rmse             1238.95
test rmse_pcutoff         NaN
test mAP                 0.00
test mAR                 0.00
Name: (0.95, 1, 10, -1, 0.6), dtype: float64
```
In the predicted poses csv, these are some likelihood values
0.029221
0.030457
0.029617
0.030167
0.030945
0.031036
0.03125


Here is the full csv:

[cage 7-2DLC_Resnet50_DLC_NewProject_GRP3_HealthyAug15shuffle1_snapshot_best-10.csv](https://github.com/user-attachments/files/21843278/cage.7-2DLC_Resnet50_DLC_NewProject_GRP3_HealthyAug15shuffle1_snapshot_best-10.csv)


Please let me know what I am doing wrong here. Thank you so much.

**Comment by maximpavliv** [comment]
Hi @NabilSNaS ,

This is indeed strange. 

Could you please check that the labels in your project have been correctly saved (use the `deeplabcut.check_labels()` method for this, see [here](https://deeplabcut.github.io/DeepLabCut/docs/standardDeepLabCut_UserGuide.html#e-check-annotated-frames))? You can post a couple of labeled images here, so I can have a look too.

If this stage works fine, can you try to train the [openfield-Pranav-2018-10-30 demo project](https://github.com/DeepLabCut/DeepLabCut/tree/main/examples/openfield-Pranav-2018-10-30) included in the DeepLabCut repository? You can simply run the [`testscript_openfielddata.py` script](https://github.com/DeepLabCut/DeepLabCut/blob/main/examples/testscript_openfielddata.py), it will create and train a model with the same architecture as yours. You can alternatively use the [Jupyter notebook](https://github.com/DeepLabCut/DeepLabCut/blob/main/examples/JUPYTER/Demo_labeledexample_Openfield.ipynb). Can you then post the resulting _train.txt_ script (you'll find it in _openfield-Pranav-2018-10-30/dlc-models-pytorch/iteration-0/openfieldOct30-trainset95shuffle13/train/_ ).

**Comment by maximpavliv** [comment]
Hi @NabilSNaS,

If the checks I suggested earlier don’t resolve the issue, could you please upload your full project folder (zipped) to Google Drive, Dropbox, or a similar service and share a link here?

That way I can try running the project on my side and investigate why the training doesn’t seem to work.

Thanks!

**Comment by maximpavliv** [comment]
Closing as stale
